libinitializer/BoostLogInitializer.cpp

```diff
@@ -61,8 +61,13 @@ void LogInitializer::initLog(
             boost::log::keywords::channel = channel));
     boost::shared_ptr<sink_t> sink(new sink_t(backend));
     /// set rotation size
-    uint64_t rotation_size = pt.get<uint64_t>(logType + ".MaxLogFileSize", 209715200);
+    uint64_t rotation_size = pt.get<uint64_t>("log.MaxLogFileSize", 209715200);
     sink->locked_backend()->set_rotation_size(rotation_size);
+
+    /// set auto-flush according to log configuration
+    bool need_flush = pt.get<bool>("log.FLUSH", true);

```

Comment:
Flush

Reply:
All configurations related to log has been modified to low case.